integration_tests/test_rough_align_matlab.py

```diff
@@ -30,6 +31,12 @@
 import shutil
 import numpy as np
 
+# skip these tests if not explicitly asked for
+pytestmark = pytest.mark.skipif(not test_legacy_rough, reason=(
```

Comment:
my first time seeing `pytest.mark.skipif`. This way for the module vs. decorating each function seems a little non-obvious, but... it seems to be how it's done